nodeEditor/src/sharedComponents/textInputLineComponent.tsx

```diff
@@ -61,7 +68,7 @@ export class TextInputLineComponent extends React.Component<ITextInputLineCompon
         this._localChange = true;
         const store = this.props.value !== undefined ? this.props.value : this.props.target[this.props.propertyName!];
 
-        if (this.props.validator && raisePropertyChanged) {
+        if (this.props.validator) {

```

Comment:
why removing raisePropertyChanged?

Reply:
to get the pop up showing again. I was actually wondering if I should remove the lines a bit lower that you added back in july since it will now do the same thing.

Reply:
actually this should work

Reply:
actually no: we do not want to validate on every change (only on blur) else we will prevent users to enter values (like the validator is that you cannot enter "var" but you want to type "variant". If you validate per stroke you will never be able to enter variant

Reply:
the code you have below does similar should i remove that segment as well then?

Reply:
I did not check but my ubber point is to make sure that your fix works with a variable that you want to name uv1 for instance. The system should block uv but not uv1

Reply:
Changed :)